package.json

```diff
@@ -79,7 +79,7 @@
     "tsconfig-paths-webpack-plugin": "^3.2.0",
     "tslib": "^1.9.3",
     "tslint": "^5.20.1",
-    "typescript": "^3.7.2",
+    "typescript": "^3.8.3",
```

Comment:
Was there a need to bump the minimum version of typescript here?

Reply:
I'm not sure of your normal process here, sorry.

There are a few ts 3.8-specific features I'd like to play with, and I guess it makes sense to document that 3.8 is the minimum. 

Happy to do whatever your team prefers.

Reply:
If we need something from 3.8.x, then bumping is fine. If there aren't any changes that require the new version, then bumping when not needed normally makes updating other libraries harder.

Reply:
From a technical standpoint it could be `^3.0.0` and it would still mean the same thing. Nobody pulls the monorepo in as a library so it's for our own uses only.

Keeping the version specifiers in sync with our other libraries sounds like a good idea.

Reply:
Sigh, I did mention that and then somehow lost it when I was rewording :( It still has to be a at least a specific version, otherwise things wouldn't compile (in this case 3.7 changed a few things so that's why it was the minimum version IIRC).